--- a/Makefile.in-org	2020-04-01 13:43:32.813637000 -0700
+++ b/Makefile.in	2020-04-01 13:44:25.257831365 -0700
@@ -457,7 +457,7 @@
 MAINTAINERCLEANFILES = Makefile.in aclocal.m4 configure config-h.in stamp-h.in config.cache config.h.in config.log config.status
 
 # Process the following directories
-SUBDIRS = config include src tests tools doc python
+SUBDIRS = config include src tests tools python
 all: all-recursive
 
 .SUFFIXES:
